src/applications/hca/components/ServiceConnectedPayConfirmation.jsx

```diff
@@ -1,70 +1,49 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import ProgressButton from 'platform/forms-system/src/js/components/ProgressButton';
 
-import ProgressButton from '~/platform/forms-system/src/js/components/ProgressButton';
-
-const ServiceConnectedPayConfirmation = ({ goBack, goForward }) => {
-  const FormNavButtons = () => (
+const ServiceConnectedPayConfirmation = ({ goBack, goForward }) => (
+  <va-alert
+    class="vads-u-margin-x--neg2p5 vads-u-margin-top--2p5"
+    status="info"
+    background-only
+  >
+    <h3 className="vads-u-margin-top--0">
+      Confirm that you receive service-connected pay for a 50% or higher
+      disability rating.
+    </h3>
+    <p>
+      You selected that you currently receive service-connected disability pay
+      for a 50% or higher disability rating. This means that you meet one of our
+      eligibility criteria and we don’t need to ask your questions about other
+      criteria, like income and military service.
+    </p>
     <div className="row form-progress-buttons schemaform-buttons">
-      <div className="small-6 medium-5 columns">
+      <div className="small-5 medium-4 columns">
         {goBack && (
           <ProgressButton
-            // eslint-disable-next-line react/jsx-no-bind
+            buttonClass="hca-progress-button usa-button-secondary"
             onButtonClick={goBack}
             buttonText="Back"
```

Comment:
I'm curious if design has looked at this... it seems that the combo should be confirm & cancel (instead of back)?

Reply:
Our UX team works closely with Danielle and others about these bits. It's been been `Back` and `Confirm` since this alert has been in the form.